src/MonitorProfiler/Logging/DebugLogger.h

```diff
@@ -17,9 +18,17 @@ class DebugLogger final :
 private:
     const static int s_nMaxEntrySize = 1000;
 
+    LogLevel m_level = LogLevel::None;
```

Comment:
Should we default this to error?

Reply:
Sure? It doesn't matter here because it gets set to something else in the ctor, either what the user specified or falls back to Information.

Reply:
Sure? It doesn't matter here because it gets set to something else in the ctor, either what the user specified or falls back to Information.